common/persistence/nosql/nosqlplugin/cassandra/tasks.go

```diff
@@ -70,13 +70,43 @@ func (db *cdb) SelectTaskList(ctx context.Context, filter *nosqlplugin.TaskListF
 		TaskListName: filter.TaskListName,
 		TaskListType: filter.TaskListType,
 
-		TaskListKind:    taskListKind,
-		LastUpdatedTime: lastUpdatedTime,
-		AckLevel:        ackLevel,
-		RangeID:         rangeID,
+		TaskListKind:            taskListKind,
+		LastUpdatedTime:         lastUpdatedTime,
+		AckLevel:                ackLevel,
+		RangeID:                 rangeID,
+		AdaptivePartitionConfig: toTaskListPartitionConfig(tlDB["adaptive_partition_config"]),
 	}, nil
 }
 
+func toTaskListPartitionConfig(v interface{}) *persistence.TaskListPartitionConfig {
+	if v == nil {
+		return nil
+	}
+	partition := v.(map[string]interface{})
```

Comment:
why not put this as the arg for v and not use `interface{}` ?

Reply:
more straightforward to handle `nil` case, which will be common.